### DISCUSSION TITLE
Looking for User Feedback

### DISCUSSION THREAD

**Discussion by ddasutein** [ddasutein/AutoRename#25] [Announcements]
Hi everyone, I am opening up a feedback form to take in some ideas and suggestions, as well as learning how some of you use my extension. I started this project almost 3 years ago and it started as a simple idea of solving a problem I had on Twitter. I follow a lot of japanese voice actresses on Twitter and admittedly, I do save these pictures. The problem I had is that when i need to recall a specific picture, i can't find the original source. It's difficult to find any old tweets on Twitter since there are limitations. Now, this extension has increased in complexity and it does a lot more than just Twitter. 

Also another fun fact is that this project happened to start right after I learned JavaScript at the company i did my internship in. In fact, to this very day as of writing this, I am still employed at that very same company. As you can imagine, this project is very special to me. Not just that but this is also the first time I published software on a storefront. Trust me, I have a lot of coding projects on my hard drive that will never see the light of day.

**Anyways, the two key areas of feedback focus are:**

1. I want to know from a regular user's perspective on why this extension is useful to you and what features do you want me to implement
2. If you use my extension for work, is there any specific feature or automation you want me to implement to make your job easier? 

### [Feedback Form Here ➡️](https://forms.gle/G5hKReRatbkZtkgFA)

If you have a GitHub account, you can just comment below. For those that don't have one can just answer the survey above!

Thank you so much! 😊

**Comment by pacman00700** [comment]
Hi,
Thanks for creating this extension.
One suggestion I have is to allow customization for any site. As of now I see you can only autorename on the sites that you, the programmer, chose.
Regards

**Reply by ddasutein** [reply]
> Hi, Thanks for creating this extension. One suggestion I have is to allow customization for any site. As of now I see you can only autorename on the sites that you, the programmer, chose. Regards

That is something planned in the future. The challenge is trying to figure out a design that can work on any site.
